dubbo-common/src/main/java/org/apache/dubbo/common/event/DubboApplicationMulticasterRegistry.java

```diff
@@ -14,24 +14,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.dubbo.metrics;
+package org.apache.dubbo.common.event;
 
 import org.apache.dubbo.common.beans.factory.ScopeBeanFactory;
-import org.apache.dubbo.metrics.event.MetricsDispatcher;
 import org.apache.dubbo.rpc.model.ApplicationModel;
 import org.apache.dubbo.rpc.model.FrameworkModel;
 import org.apache.dubbo.rpc.model.ModuleModel;
 import org.apache.dubbo.rpc.model.ScopeModelInitializer;
 
-public class MetricsScopeModelInitializer implements ScopeModelInitializer {
+/**
+ * Initialize {@link DubboLifecycleEventMulticaster} for {@link ApplicationModel}
+ *
+ * @see DubboLifecycleEventMulticaster
+ * @since 3.3.0
+ */
+public class DubboApplicationMulticasterRegistry implements ScopeModelInitializer {
```

Comment:
DubboApplicationMulticasterRegistry can have more specifications and make it more general. DubboApplicationMulticasterRegistry needs to be re-named.

Reply:
Rename DubboApplicationMulticasterRegistry to DubboMulticasterScopeModelInitializer.
Mark DubboEventMulticaster as SPI to allow extension implementation.

Reply:
I'm not sure what you mean. Does this meet your expectations?